### ISSUE TITLE
Test dropout

### ISSUE THREAD

**Issue by jeohalves** [open-mmlab/mmdetection#7837]
I'm trying to employ a Monte Carlo Dropout and i'm curious on how to set dropout in the testing phase. Is there any way to configure test dropout in MMDetection? I'm using a MaskRCNN with Swin Transformer (as backbone).

**Comment by ZwwWayne** [comment]
Hi @jeohalves ,
Which layer you want to apply dropout to? If it is in backbone, you can try to add it add it as plugin. See examples in https://github.com/open-mmlab/mmdetection/blob/master/configs/empirical_attention/faster_rcnn_r50_fpn_attention_0010_1x_coco.py. You might need to implement a new Dropout layer that dropout at test, since I remember the PyTorch dropout will do nothing when `self.training=False`
